cmd/podman/system/connection/list.go

```diff
@@ -114,23 +120,31 @@ func inspect(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if format != "" {
-		rpt, err = rpt.Parse(report.OriginUser, format)
-	} else {
+	switch format {
+	case "tls":
```

Comment:
There’s some precedent to special-casing `--format=…` values (in particular we accept --format json` in places), and to an extent I appreciate the ergonomic, but, on the _principle_ of the thing, I personally dislike values that have two different semantics. If `--format hello` outputs literal "hello", it’s inconsistent that `--format tls` does not output literal "tls".

I’m not authoritative for Podman, and note that this _was_ pre-approved in https://github.com/containers/podman/pull/24601#discussion_r2270716090 .